bleak/backends/corebluetooth/client.py

```diff
@@ -48,9 +48,7 @@ def __init__(self, address_or_ble_device: Union[BLEDevice, str], **kwargs):
 
         if isinstance(address_or_ble_device, BLEDevice):
             self._device_info = address_or_ble_device.details
-            self._central_manager_delegate = address_or_ble_device.metadata.get(
```

Comment:
I generally always use the `get` method for dict value retrieval, but here the `delegate` key is strictly needed for the functionality is it not?

Reply:
Correct. That is why I changed it here.